saleor/dashboard/product/views.py

```diff
@@ -80,10 +82,15 @@ def product_class_delete(request, pk):
                 'Dashboard message',
                 'Deleted product type %s') % product_class)
         return redirect('dashboard:product-class-list')
-    return TemplateResponse(
-        request,
-        'dashboard/product/product_class/modal_confirm_delete.html',
-        {'product_class': product_class, 'products': products})
+
+    products = [str(p) for p in product_class.products.all()]
```

Comment:
I think you don't need this iteration over products here. If you want to display products in a template you can safely pass the queryset without any preprocessing; iterating over products in a template will do the same what `str(product)` does. But from what I see you're passing products to the context but they are not rendered in the template.

Reply:
Just moved this line closer to usage :) You're right anyway, I'll change it.